### ISSUE TITLE
Timeout with Get-ASCDataCollection

### ISSUE THREAD

**Issue by kmchannon** [Microsoft/Azure-Security-Center#2]
Get-ASCDataCollection : {"error":{"code":"GatewayTimeout","message":"The gateway did not receive a response from 'Microsoft.Security' within the specified time 
period."}}
At line:1 char:1
+ Get-ASCDataCollection -ComputeType Compute -ResourceGroup NifastNAVDR ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Get-ASCDataCollection

**Comment by Javanite** [comment]
Hi Kmchannon,

I just confirmed this behavior in my tenant as well. I'm not sure this is an issue with the module, and may be a problem with the service itself. I have looped in ASC engineering for a response and will update this issue thread when I have more information. I apologize for the inconvenience.

Edit:

After talking with engineering, they are decommissioning the dataCollectionResults endpoint. This means the Get-ASCDataCollection cmdlet is no longer usable. I will remove it from the module.
